tests/TestSearchPath.cc

```diff
@@ -17,55 +17,41 @@
 
 #include <catch2/catch_test_macros.hpp>
 
-#include <stdlib.h>
 #include <string>
 #include <vector>
 
-#include "generated/tests_config.hh"
-
 #include "SearchPath.hh"
-#include "Utilities.hh"
 
-TEST_CASE("Check SearchPath honors XDG_DATA_HOME", "[SearchPath]") {
-    setenv("XDG_DATA_HOME", "/usr/share", 1);
-    setenv("XDG_DATA_DIRS", " ", 1);
+static constexpr auto always_exists = [](const std::string &) { return true; };
```

Comment:
I'd use a normal function here instead of fancy lambdas.

Reply:
Makes sense, done.